### DISCUSSION TITLE
Treeview : use open with return-object

### DISCUSSION THREAD

**Discussion by ThomasKientz** [vuetifyjs/vuetify#11490] [General] [answered]
Hi guys,

Is it possible to use the `open` option with `return-object` option on a `Treeview` ? 

https://codepen.io/thomaskientz/pen/xxwevYP

**Comment by nekosaur** [comment, marked as answer]
No. Currently (as of version `2.2.29`) `return-object` on affects `active` and `value` (selected). You can create a feature request if you feel like this is something that should exist.

**Comment by ThomasKientz** [comment]
Feature request : https://github.com/vuetifyjs/vuetify/issues/11500
